backend/cmd/start.go

```diff
@@ -74,10 +75,14 @@ const (
 	// defaultEtcdName is the default etcd member node name (single-node cluster
 	// only)
 	defaultEtcdName = "default"
-	// DefaultEtcdPeerURL is the default URL to listen for Etcd peers (single-node
+	// defaultEtcdPeerURL is the default URL to listen for Etcd peers (single-node
 	// cluster only)
 	defaultEtcdPeerURL = "http://127.0.0.1:2380"
 
+	// defaultEtcdAdvertiseClientURL is the default list of this member's client
+	// URLs to advertise to the rest of the cluster
+	defaultEtcdAdvertiseClientURL = "http://localhost:2379"
```

Comment:
This is `localhost` intentionally since this field can accept hostnames?

Reply:
Yep, and it's etcd default value too (https://coreos.com/etcd/docs/latest/op-guide/configuration.html)